src/components/scenes/WalletListScene.js

```diff
@@ -465,7 +467,12 @@ export default class WalletList extends Component<Props, State> {
         noButtonText: s.strings.string_cancel_cap,
         yesButtonText: s.strings.title_create_wallet
       })
-      return (await showModal(modal)) ? Actions[Constants.CREATE_WALLET_SELECT_CRYPTO]() : null
+
+      if (this.props.ethereumWalletType) {
+        return (await showModal(modal)) ? Actions[Constants.CREATE_WALLET_SELECT_FIAT]({ selectedWalletType: this.props.ethereumWalletType }) : null
+      } else {
+        return Alert.alert(s.strings.create_wallet_invalid_input, s.strings.create_wallet_select_valid_crypto)
```

Comment:
Why would you ever get into this `else` case?

Reply:
I checked the code on the create wallet screen and somehow there is a scenario that it can be not supported?

I am really not sure so just to be safe I just copied the code. If the ethereum wallet is always supported then no problem ill remove even the null check.

Reply:
Ok. let's leave it for now.